sql/ddl/indices/create-indexes.md

```diff
@@ -56,44 +56,44 @@ There are two main types of indices:
 - **clustered** - they store the actual data of the whole element's row. An important feature of the clustered indices is that the indexed values are sorted in either ascending or descending order. This means that a table can only have one clustered index on it
 - **non-clustered** - they store only values from the indexed columns and the row locators to the actual data on the row. A table can have multiple non-clustered indices associated with it
 
-Note that a table with an clustered index on it is called *"clustered table"* and one with non-clustered only is called *"heap"*.
+Note that a table with an clustered index on it is called *"clustered table"* and one that contains non-clustered indices only is called *"heap"*.
```

Comment:
No need to quote these names, the italics already emphasize them enough. Also added a grammar fix for "a" instead of "an"

```suggestion
Note that a table with a clustered index on it is called *clustered table* and one that contains non-clustered indices only is called *heap*.
```

Reply:
Also, in most of the text bellow use use **clustered index** over *clustered index* so we should probably unite those (I personally prefer using emphasized over bold but not a big deal either way)